.circleci/config.yml

```diff
@@ -23,36 +23,42 @@ jobs:
       - checkout
       - setup_remote_docker:
           version: 20.10.11
-          # docker_layer_caching is behind a paywall so we do it manually below
           # docker_layer_caching: true # DLC will explicitly cache layers here and try to avoid rebuilding.
+          # docker_layer_caching is behind a paywall so we do it manually below
+          # based on: https://anonoz.github.io/tech/2018/06/17/circleci-docker-caching.html
+      - run:
+          name: Check if there are newer images on Docker Hub
+          command: |
+              REPO=elabftw/elabimg
+              TAG=hypernext
+              TOKEN=$(curl -s "https://auth.docker.io/token?service=registry.docker.io&scope=repository:$REPO:pull" | jq -r .token)
```

Comment:
wouldn't it be easier/more robust to define a pull token in env instead?

Reply:
Yes that would be great.
I initially just followed the blueprint but as nearly always there is room for improvement.
On the other hand I'm not sure if it is really faster.

Reply:
You can use DOCKERHUB_PULL_TOKEN now.

Reply:
Fun fact: the other token present in Docker hub was also created on April 20th (last year)! Guess it's a good day to create a token :p